cli_tools/gce_ovf_import/README.md

```diff
@@ -93,33 +96,50 @@ go get github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_ovf_impo
 + `-disable-stdout-logging` do not display individual workflow logs on stdout
 + `-node-affinity-label` Node affinity label used to determine sole tenant node to schedule this instance on. Label is of the format: <key>,<operator>,<value>,<value2>... where <operator> can be one of: IN, NOT. For example: workload,IN,prod,test is a label with key 'workload' and values 'prod' and 'test'. This flag can be specified multiple times for multiple labels.
 + `-release-track` Release track of OVF import. One of: %s, %s or %s. Impacts which compute API release track is used by the import tool.
++ `-hostname` Specify the hostname of the instance to be created. The specified hostname must be RFC1035 compliant.
++ `-machine-image-storage-location` GCS bucket storage location of the machine image being imported (regional or multi-regional). Applicable only if `-machine-image-name` is provided.
```

Comment:
It's long and not that clear. How about machine-image-output-uri?

Reply:
This is not an URI but storage location, like a region or a multi-region. Equivalent for images is --storage-location. Since using that name would cause confusion in the context of ovf improt (since it imports instances), -machine-image-storage-location accurately describes what this flag is. From gcloud perspective, the flag will be --storage-location since the context will be machine images.